pumps/hybrid.go

```diff
@@ -128,8 +130,25 @@ func (p *HybridPump) Init(config interface{}) error {
 			p.trackAllPaths = trackAllPaths.(bool)
 		}
 
-		if storeAnalyticPerMinute, ok := meta["store_analytics_per_minute"]; ok {
-			p.storeAnalyticPerMinute = storeAnalyticPerMinute.(bool)
+		if storeAnalyticsPerMinute, ok := meta["store_analytics_per_minute"].(bool); ok {
+			p.storeAnalyticsPerMinute = storeAnalyticsPerMinute
+			p.aggregationTime = 1
+		} else {
+			aggregationTime, ok := meta["aggregation_time"].(int)
+			if !ok {
+				p.log.Warn("aggregation_time is not configured. The default value will be set to 60")
```

Comment:
lets put something like: `p.log.Warn("aggregation_time is not configured. The default value will be used (60 minutes)")`

or something similar